scripts/validate_cclw_sheet/validate-ingest.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+#
+# Posts the request to validate the ingest.
+#
+# 
+set -eou pipefail
+
+CSV_FILE=$1
+USER=${SUPERUSER_EMAIL:-user@navigator.com}
+PASSWORD=${SUPERUSER_PASSWORD:-password}
+ 
+# ---------- Functions ----------
+
+wait_for_server() {
+    printf 'Waiting for server.'
+    until $(curl --output /dev/null --silent  ${TEST_HOST}/health); do
+        printf '.'
+        sleep 1
+    done
+    echo
+}
+
+get_token() {
+    curl -s \
+    -H "Content-Type: application/x-www-form-urlencoded" \
+    -d "username=${USER}&password=${PASSWORD}" \
+        ${TEST_HOST}/api/tokens | \
+        jq ".access_token" | tr -d '"'
+}
+
+validate_csv() {
+    TOKEN=$(get_token)
+    curl -s \
+        -H "Authorization: Bearer ${TOKEN}" \
+        -F "law_policy_csv=@${CSV_FILE}" \
+        ${TEST_HOST}/api/v1/admin/bulk-ingest/validate/cclw/law-policy | jq
+}
+
+echo "Validating as ${USER}"
+wait_for_server
```

Comment:
Do we need a catch here for if the wait hangs?

Reply:
I'm relying on GA to do this for us :smile: